Currently, makecert can make a cert using sha256

TaSato-MSFT opened the pull request:
makecert.exe(Version 6.1.7600.16385 and above) can make a certificate using sha256(sha2) above.

Windows 7 93184 - makecert needs to support SHA2 certificates
http://bugcheck/bugs/Windows7/93184

So, it is wrong that the original sentence of "However, MakeCert can't generate SHA-2 certificates, only SHA-1."
This makes a customer confusing.

azurecla commented:
Hi __@AzureDocsBot__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

jodoglevy commented:
Please ignore - testing for HackADoc.

TaSato-MSFT committed:
Currently, makecert can make a cert using sha256

makecert.exe(Version 6.1.7600.16385 and above) can make a certificate using sha256(sha2) above.

Windows 7 93184 - makecert needs to support SHA2 certificates
http://bugcheck/bugs/Windows7/93184

So, it is wrong that the original sentence of "However, MakeCert can't generate SHA-2 certificates, only SHA-1."
This makes a customer confusing.
